requirements/base.txt

```diff
@@ -1,2 +1,9 @@
-# Base requirements are empty. All of our minimum requirements are specified in setup.py's install_requires variable!
--e .
+django-filter==0.13.0
+django-js-reverse==0.7.2
+six==1.10.0
+djangorestframework==3.3.3
+django==1.9.7
+colorlog==2.6.0
+docopt==0.6.2
+django-mptt==0.8.5
+https://github.com/cherrypy/cherrypy/archive/v6.2.0.zip
```

Comment:
![image](https://user-images.githubusercontent.com/374612/27456846-c4d3df7c-57a2-11e7-9e4c-1decc66e2487.png)

This is great, now we can get rid of it, time flies, things that are annoying are fixed, my belief in the open source python world is at a new high :)